src/mic-recorder.js

```diff
@@ -90,10 +90,10 @@ class MicRecorder {
     this.config.sampleRate = this.context.sampleRate;
     this.lameEncoder = new Encoder(this.config);
 
-    const audio = { deviceId: { exact: this.config.deviceId } };
+    const audio = this.config.deviceId ? { deviceId: { exact: this.config.deviceId } } : true;
 
     return new Promise((resolve, reject) => {
-      navigator.mediaDevices.getUserMedia({ audio })
+      navigator.mediaDevices.getUserMedia({ audio: audio })
```

Comment:
This is unnecessary.

Reply:
Not quite sure what you mean sorry, which part is unnecessary?

Reply:
You're adding `: audio` to this PR, but this is a shorthand notation, thus it's not necessary.

Reply:
I found i was getting a Type error when specifying a deviceId and not using `audio: audio`

Reply:
I mean that `{ audio }` and `{ audio: audio }` are exactly the same.

Reply:
I need to specify either `audio: true` or `audio: { deviceId : { exact: this.config.deviceId } }` latest commit makes it a bit more clear

Reply:
omg sorry i didnt read the link, i get what you mean now haha